### PR TITLE
LPS-70349 Upgrade CXF modules to latest version

### PR DESCRIPTION
**Pull request by csierra**

### CODE REVIEW

**Comment by csierra** [comment]
hey @topolik, 
I know we said to put this off, but I already had done this already so I did not want to waste it. I tested it against adaptive-media and our basic tests.

File: modules/apps/foundation/portal-remote/portal-remote-cxf-jaxrs-common/build.gradle [hunk 1]
```diff
@@ -3,9 +3,10 @@ dependencies {
 	provided group: "javax.json", name: "javax.json-api", version: "1.0"
 	provided group: "javax.validation", name: "validation-api", version: "1.1.0.Final"
 	provided group: "javax.ws.rs", name: "javax.ws.rs-api", version: "2.0.1"
-	provided group: "org.apache.cxf", name: "cxf-core", version: "3.0.3"
-	provided group: "org.apache.cxf", name: "cxf-rt-frontend-jaxrs", version: "3.0.3"
-	provided group: "org.apache.cxf", name: "cxf-rt-rs-extension-providers", version: "3.0.3"
+	provided group: "org.apache.cxf", name: "cxf-core", version: "3.1.9"
+	provided group: "org.apache.cxf", name: "cxf-rt-frontend-jaxrs", version: "3.1.9"
+	provided group: "org.apache.cxf", name: "cxf-rt-rs-extension-providers", version: "3.1.9"
+	provided group: "org.apache.cxf", name: "cxf-rt-rs-json-basic", version: "3.1.9"
```
Comment (topolik) [inline modules/apps/foundation/portal-remote/portal-remote-cxf-jaxrs-common/build.gradle]: Hi @csierra,

the line with json-basic is new. Is this correct?

Comment (csierra) [inline modules/apps/foundation/portal-remote/portal-remote-cxf-jaxrs-common/build.gradle]: yup... it should... it seems they changed the packages topology with the new versions and they extracted that to a new package...

Comment (topolik) [inline modules/apps/foundation/portal-remote/portal-remote-cxf-jaxrs-common/build.gradle]: ok ,thanks!

**Comment by topolik** [comment]
Pull request submitted at: https://github.com/brianchandotcom/liferay-portal/pull/46837

View just my changes: https://github.com/topolik/liferay-portal/compare/csierra:c883146578...99e2868d2e
View total diff: https://github.com/topolik/liferay-portal/compare/a12573ded1...99e2868d2e

**Pull request closed without merging**